updater/bin/update_files.rb

```diff
@@ -31,7 +32,12 @@ class UpdaterKilledError < StandardError; end
       Dependabot::Environment.job_id,
       Dependabot::Environment.job_token
     )
-  Dependabot::Service.new(client: api_client).capture_exception(error: error, tags: tags)
+  service = Dependabot::Service.new(client: api_client)
+
+  Dependabot::ExceptionCapturer.handle_captured_exceptions do |exception|
+    service.capture_exception(error: exception, tags: tags)
+  end
+  service.capture_exception(error: error, tags: tags)
```

Comment:
I think we want to put this down below 👇 We're eating the exception in the MessageBuilder so it won't reach this code in the trap.

What might be simpler would be give the ExceptionCapturer the service here (outside of the trap), and have it log to the API immediately when `capture_exception` is called.